Tools/scripts/generate_manifest.py

```diff
@@ -413,7 +413,7 @@ def add_firmware_data_from_dir(self,
 
                 filepath = os.path.join(some_dir, filename)
                 firmware_format = self.firmware_format_for_filepath(filepath)
-                if firmware_format not in [ "ELF", "abin", "apj", "hex", "px4", "bin" ]:
+                if firmware_format not in [ "elf", "ELF", "abin", "apj", "hex", "px4", "bin" ]:
```

Comment:
```suggestion
                if firmware_format.lower() not in [ "elf", "abin", "apj", "hex", "px4", "bin" ]:
```
Would this also work?

Reply:
Yes it would, and I considered such a patch, and I *also* considered making a frozenset.

I went with the minimal, understandable, patch :-)